tests/pillar/under-apache.sls

```diff
@@ -0,0 +1,182 @@
+
```

Comment:
I don't know why but `under-apache` just sounds somehow creepy.. don't you want to rename it to `apache_wsgi.sls` to make more sense?